nkpremices/Banka
# #165355752 Feedback implementation (day 1)

#### What does this PR do?
Have the .gitignore and the readme files improved

#### Description of Task to be completed?
* Remove unnecessary files from the .gitignore
* Format the command line instructions on the readme

#### How should this be manually tested?
Go to the Github repo:
* Check in the readme if the command line instructions are at a new line
* Check the .gitignore file
